homeassistant/components/nws/strings.json

```diff
@@ -13,11 +13,11 @@
       }
     },
     "error": {
-      "cannot_connect": "Failed to connect, please try again",
-      "unknown": "Unexpected error"
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
+      "unknown": "[%key:common::config_flow::error::unknown%]"
     },
     "abort": {
-      "already_configured": "Device is already configured"
+      "already_configured": "[%key:common::config_flow::abort::single_instance_allowed%]"
```

Comment:
This should not use `single_instance_allowed` as the integration allows more than one config entry. Use `already_configured_service`.

Reply:
Solved in #41062